### ISSUE TITLE
InstanceAlreadyExistsException with new multi-threaded consumer

### ISSUE THREAD

**Issue by kinghuang** [SourceLabOrg/kafka-webview#175]
The new multi-threaded consumer in 2.3.0 doesn't seem to work for me. I just get unending errors in the logs for `InstanceAlreadyExistsException`. Setting `multiThreadedConsumer: false` in the config file works.

```
javax.management.InstanceAlreadyExistsException: kafka.consumer:type=app-info,id=KafkaWebViewConsumer-UserId1-web-256d1a7efb57dfed03a11f1fa57cd1dd41cc5c1b
	at com.sun.jmx.mbeanserver.Repository.addMBean(Repository.java:437) ~[na:1.8.0_151]
	at com.sun.jmx.interceptor.DefaultMBeanServerInterceptor.registerWithRepository(DefaultMBeanServerInterceptor.java:1898) ~[na:1.8.0_151]
	at com.sun.jmx.interceptor.DefaultMBeanServerInterceptor.registerDynamicMBean(DefaultMBeanServerInterceptor.java:966) ~[na:1.8.0_151]
	at com.sun.jmx.interceptor.DefaultMBeanServerInterceptor.registerObject(DefaultMBeanServerInterceptor.java:900) ~[na:1.8.0_151]
	at com.sun.jmx.interceptor.DefaultMBeanServerInterceptor.registerMBean(DefaultMBeanServerInterceptor.java:324) ~[na:1.8.0_151]
	at com.sun.jmx.mbeanserver.JmxMBeanServer.registerMBean(JmxMBeanServer.java:522) ~[na:1.8.0_151]
	at org.apache.kafka.common.utils.AppInfoParser.registerAppInfo(AppInfoParser.java:62) ~[kafka-clients-2.0.1.jar!/:na]
	at org.apache.kafka.clients.consumer.KafkaConsumer.<init>(KafkaConsumer.java:791) [kafka-clients-2.0.1.jar!/:na]
	at org.apache.kafka.clients.consumer.KafkaConsumer.<init>(KafkaConsumer.java:615) [kafka-clients-2.0.1.jar!/:na]
	at org.apache.kafka.clients.consumer.KafkaConsumer.<init>(KafkaConsumer.java:596) [kafka-clients-2.0.1.jar!/:na]
	at org.sourcelab.kafka.webview.ui.manager.kafka.KafkaConsumerFactory.createConsumer(KafkaConsumerFactory.java:65) [classes!/:2.3.0]
	at org.sourcelab.kafka.webview.ui.manager.kafka.KafkaConsumerFactory.createConsumerAndSubscribe(KafkaConsumerFactory.java:73) [classes!/:2.3.0]
	at org.sourcelab.kafka.webview.ui.manager.kafka.ParallelWebKafkaConsumer.createNewConsumer(ParallelWebKafkaConsumer.java:278) [classes!/:2.3.0]
	at org.sourcelab.kafka.webview.ui.manager.kafka.ParallelWebKafkaConsumer.lambda$consumePerPartition$0(ParallelWebKafkaConsumer.java:103) [classes!/:2.3.0]
	at java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1590) ~[na:1.8.0_151]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[na:1.8.0_151]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[na:1.8.0_151]
	at java.lang.Thread.run(Thread.java:748) ~[na:1.8.0_151]
```

**Comment by Crim** [comment]
Does consuming fail to work, or are you just seeing these errors but consuming works OK? 

When I saw those errors locally, it was because the parallel consumers were attempting to register the same JMX bean, and you'd get this error in the log.  As far as I could determine from reviewing Kafka's consumer configurations, there was no way to prevent the JMX Metric Reporter from being registered.  

Consuming would work correctly however.

**Comment by kinghuang** [comment]
You're right. I mistakenly thought consuming didn't work. But, it actually does.

**Comment by Crim** [comment]
Glad to hear it actually works.  The flood of errors are far from ideal, I'll try to put together a patch here shortly that works out a way to avoid them.

**Comment by Crim** [comment]
Fix for this should be included in upcoming 2.3.1 release.  Thanks!

**Comment by Crim** [comment]
released [2.4.0](https://github.com/SourceLabOrg/kafka-webview/releases/tag/v2.4.0) which should resolve those errors from being shown.

Thanks!
